Embedded Metadata.js

```diff
@@ -1145,12 +1148,12 @@ var testCases = [
 				"title": "Northwestern Can't Quit ASA Over Boycott Because it is Not a Member",
 				"creators": [
 					{
-						"firstName": "Eugene",
-						"lastName": "Kontorovich",
+						"firstName": "",
+						"lastName": "https://www.facebook.com/EugeneKontorovich?ref=hl",
```

Comment:
this is the only disadvantage we're getting from this. I'm unsure what to do about it--it comes from `property="article:author"`. I think that's still preferable to include, but am open for suggestions (e.g. we could test that for twitter and facebook URLs and reject it if those are in the string).